MoveBaseActionGoal goal
MoveBaseActionResult result
MoveBaseActionFeedback feedback